src/riak_core_util.erl

```diff
@@ -742,6 +787,25 @@ incr_counter(CounterPid) ->
 decr_counter(CounterPid) ->
     CounterPid ! down.
 
+multi_keydelete_test_() ->
+    Languages = [{lisp, 1958},
+                 {ml, 1973},
+                 {erlang, 1986},
+                 {haskell, 1990},
+                 {ocaml, 1996},
+                 {clojure, 2007},
+                 {elixir, 2012}],
+    ?_assertMatch(
+       [{lisp, _}, {ml, _}, {erlang, _}, {haskell, _}],
```

Comment:
Can we guarantee the order of the returned list?  If so, this is fine, otherwise we might want to sort the output before comparing the results.